contrib/python/kalman.py

```diff
@@ -0,0 +1,81 @@
+#
+# Simple Implementation of the Kalman Filter for 1D data, without any dependencies
+# Originally written in JavaScript by Wouter Bulten
+# Now rewritten in Python
+#
+# 2017
+#
+# license GNU LESSER GENERAL PUBLIC LICENSE v3
+# author Sifan Ye
+#
+# see https://github.com/wouterbulten/kalmanjs
+#
+
+import math
+
+class KalmanFilter:
+    A = 1
```

Comment:
These variables should not be declared static but should be class members. Please move them to the constructor.